edge-functions/feature-flag-apple-store/pages/api/store/close.ts

```diff
@@ -0,0 +1,26 @@
+import { set } from 'lib/upstash-redis'
+
+export const config = {
+  runtime: 'experimental-edge',
+}
+
+export default async function CloseStore() {
+  try {
+    const { result } = await set('store-closed', 'true')
+
+    if (result !== 'OK') {
+      throw `Upstash throwed an error: ${result} `
```

Comment:
This is not an error as the call to upstash worked and was correct, but something else failed if the result is not `'OK'`
```suggestion
      throw new Error(`Unexpected result from Upstash: ${result}`)
```